ledger/sandbox-on-x/BUILD.bazel

```diff
@@ -116,7 +117,7 @@ da_scala_binary(
     main_class = "com.daml.ledger.sandbox.Main",
     tags = [
         "fat_jar",
-        "maven_coordinates=com.daml:sandbox-on-x-app:__VERSION__",
+        "maven_coordinates=com.daml:sandbox-on-x-app-jar:__VERSION__",
```

Comment:
I believe you don’t actually need different maven coordinates because for the lib we append `_2.13` whereas we don’t do this for the standalone JAR. But happy either way.

Reply:
Yeah I was thinking that as well. But I prefer it like this to avoid potential headaches when inspecting the maven registry